enroll_app/rest_utils.cpp

```diff
@@ -79,8 +87,8 @@ std::string post(std::string url, std::string content)
 /*
  * KMS server post
  */
-void post_KMS(std::string url, std::string content, RetJsonObj *retJsonObj)
+void post_KMS(std::string url, std::string content, RetJsonObj *retJsonObj, bool secure)
```

Comment:
"secure" may lead some confuese, suggest to change it as verify? or something better words